development-docs/TESTING.md

```diff
@@ -206,6 +208,25 @@ On the other hand you can also override it in mvn command using additional param
 - -Djunit.jupiter.execution.parallel.enabled=true
 - -Djunit.jupiter.execution.parallel.config.fixed.parallelism=5
 
+## Adding new test suite to our system tests
```

Comment:
What about some `make` target which will check this in case of adding a new package?

Reply:
After some consideration, I have implemented a dynamic scan of all suites, which needs auxiliary namespaces...So this is not needed anymore :)